nexus/tests/integration_tests/basic.rs

```diff
@@ -54,55 +54,53 @@ async fn test_basic_failures(cptestctx: &ControlPlaneTestContext) {
             body: None,
         },
 
-        // Error case: GET /organizations/test-org/projects/nonexistent (a
-        // possible value that does not exist inside a collection that does
-        // exist) from an authorized user results in a 404.
+        // Error case: a possible value that does not exist inside a collection
+        // that does exist) from an authorized user results in a 404.
         TestCase {
             method: Method::GET,
-            uri: "/organizations/test-org/projects/nonexistent",
+            uri: "/v1/projects/nonexistent?organization=test-org",
```

Comment:
This error message is different now that we take name or ID. Can't say `data did not match any variant of untagged enum NameOrId` is particularly nice, but it does suggest that the thing you passed is not a name or ID. I'm thinking I should change the assert here to the new message and make an issue to improve it. @zephraph?

```
thread 'integration_tests::basic::test_basic_failures' panicked at 'assertion failed: `(left == right)`
  left: `"bad parameter in URL path: name must begin with an ASCII lowercase character"`,
 right: `"bad parameter in URL path: data did not match any variant of untagged enum NameOrId"`', nexus/tests/integration_tests/basic.rs:155:9
```